File: python/MASTER_SLAVE_WHOLE_PART_PROXY/whole_part/DatosCliente.py
class DatosCliente:
    nombres = None
    apellidoPaterno = None
    apellidoMaterno = None
    edad = None
    estudiante = None
    clienteEspecial = None
    
    def setDatosCliente(self, nombres,
                        apellidoPaterno,
                        apellidoMaterno,
                        edad,
                        estudiante,
                        clienteEspecial):
        self.nombres = nombres
        self.apellidoPaterno = apellidoPaterno
        self.apellidoMaterno = apellidoMaterno
        self.edad = edad
        self.estudiante = estudiante
        self.clienteEspecial = clienteEspecial
        
    def getDatosCliente(self):
        
        assert (self.nombres != None and 
                self.apellidoPaterno != None and 
                self.apellidoMaterno != None and
                self.edad != None and
                self.estudiante != None and
                self.clienteEspecial != None )
        
        return (self.nombres,
               self.apellidoPaterno,
               self.apellidoMaterno,
               self.edad,
               self.estudiante,
               self.clienteEspecial)

File: php/OBJECT_ADAPATER_CLASS_ADAPTER_WHOLE_PART/datos/DatosCliente.php
<?php

class DatosCliente {

    public $nombre;
    public $apellidoPaterno;
    public $apellidoMaterno;
    public $tipoCliente;
    public $edad;


    public function setDatosCliente($nombre, $apellidoP, $apellidoM, $edad, $tipo) {
        $this->nombre = $nombre;
        $this->apellidoPaterno = $apellidoP;
        $this->apellidoMaterno = $apellidoM;
        $this->edad = $edad;
        $this->tipoCliente = $tipo;
    }
    public function getNombre() {
        return $this->nombre;
    }

    public function getApellidoPaterno() {
        return $this->apellidoPaterno;
    }

    public function getApellidoMaterno() {
        return $this->apellidoMaterno;
    }

    public function getTipoCliente() {
        return $this->tipoCliente;
    }

    public function getEdad() {
        return $this->edad;
    }

    public function setNombre($nombre): void {
        $this->nombre = $nombre;
    }

    public function setApellidoPaterno($apellidoPaterno): void {
        $this->apellidoPaterno = $apellidoPaterno;
    }

    public function setApellidoMaterno($apellidoMaterno): void {
        $this->apellidoMaterno = $apellidoMaterno;
    }

    public function setTipoCliente($tipoCliente): void {
        $this->tipoCliente = $tipoCliente;
    }

    public function setEdad($edad): void {
        $this->edad = $edad;
    }



}
